File: js/src/formulas/sigmaSquare.ts
export default function sigmaSquare (n: number, k: number) {
  return (((1 / 6) * n * (n + 1) * ((2 * n) + 1)) - ((1 / 6) * k * (k + 1) * ((2 * k) + 1)))
}

File: cs/src/formulas/sigmaSquare.cs
public static class SigmaSquare
{
    public static double Calculate(double n, double k)
    {
        return (1.0 / 6.0) * n * (n + 1) * (2 * n + 1) - (1.0 / 6.0) * k * (k + 1) * (2 * k + 1);
    }
}
